processor/tailsamplingprocessor/config.go

```diff
@@ -62,6 +64,33 @@ type SubPolicyCfg struct {
 	StringAttributeCfg StringAttributeCfg `mapstructure:"string_attribute"`
 	// Configs for rate limiting filter sampling policy evaluator.
 	RateLimitingCfg RateLimitingCfg `mapstructure:"rate_limiting"`
+	// Configs for latency filter sampling policy evaluator.
+	LatencyCfg LatencyCfg `mapstructure:"latency"`
```

Comment:
Latency and status code are now duplicated.

Reply:
my bad, I messed up with merging the changes from `main`